community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -472,6 +485,129 @@ def update_topology(lkp: util.Lookup) -> None:
         log.debug("Topology configuration updated. Reconfiguring Slurm.")
         util.scontrol_reconfigure(lkp)
 
+
+def run_command(cmd: str, err_msg: str = None) -> subprocess.CompletedProcess:
+    res = subprocess.run(cmd, shell=True, universal_newlines=True, check=False,
+                         stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    if res.returncode != 0:
+        raise subprocess.SubprocessError(f"{err_msg}:\n{res.stderr}")
+
+    return res
+
+def delete_reservations(reservation_map: Dict[str, str]) -> None:
```

Comment:
Is there a particular reason to use `gcloud` instead of REST API?

Reply:
Removed gcloud.